tools/reporter/config.js

```diff
@@ -0,0 +1,6 @@
+'use strict'
+
+module.exports = {
```

Comment:
I use convict to manage configurations. It has a nice feature that allows you to override values per environment but also override them via env vars and I assume that this is going to end up running in a docker container. Just an idea.

Reply:
this config uses env vars too, not sure if convict will be an overkill or not

Reply:
yeah, if it uses env vars should be fine.